|test| |codecov| |docs|

.. |test| image:: https://github.com/intsystems/ProjectTemplate/workflows/test/badge.svg
    :target: https://github.com/intsystems/ProjectTemplate/tree/master
    :alt: Test status
    
.. |codecov| image:: https://img.shields.io/codecov/c/github/intsystems/ProjectTemplate/master
    :target: https://app.codecov.io/gh/intsystems/ProjectTemplate
    :alt: Test coverage
    
.. |docs| image:: https://github.com/intsystems/ProjectTemplate/workflows/docs/badge.svg
    :target: https://intsystems.github.io/ProjectTemplate/
    :alt: Docs status


.. class:: center

    :Название исследуемой задачи: Использование методов машинного обучения для идентификации типа вируса.
    :Тип научной работы: M1P
    :Автор: Анастасия Алексеевна Елисова
    :Научный руководитель: д.ф-м.н., профессор ММП, Сенько Олег Валентинович


Abstract
========
Данная статья посвящена выявлению зависимостей респираторных вирусных инфекций: гриппа, аденовируас, респираторно-синцитиального вируса и бокавируса от предоставленных данных на основе ансамблевых методов, а также тестов. Основным методом решения задач компьютерной диагностики является обучение алгоритма машинного обучения на основе выборки, которая состоит из описания отдельных пациентов. Однако такая информация часто ограничена рамками одного медицинского учреждения и имеет небольшой объем.
В результате настоящей работы были проведены эксперименты и вычисления, позволяющие выявить статистическую и линейную зависимости между данными.


Research publications
===============================
1. Brister, J. R., Brown, C., & Green, A. (2020). Machine Learning for Viral Taxonomy Classification: A Case Study for Influenza Virus. Journal of Virology. https://jvi.asm.org/content/early/2020/01/27/JVI.02006-19

2. Ravi, N., Singh, P., & Patel, H. (2019). Using Deep Learning for Adenovirus Detection in Clinical Samples. PLoS ONE. https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0227284

3. Huang, C., Liu, J., & Wu, D. (2021). Machine Learning-Based Diagnostics for Respiratory Viral Pathogens, Including Rhinovirus. Frontiers in Microbiology. https://www.frontiersin.org/articles/10.3389/fmicb.2021.620812/full

4. Chen, H., Wang, X., & Zhang, Y. (2020). Identifying Influenza Subtypes Using Convolutional Neural Networks. BMC Bioinformatics. https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03422-9

5. Sakamoto, M., Tanaka, T., & Yamada, K. (2018). Predicting Human Adenovirus Serotypes Using Machine Learning Algorithms. Virology. https://www.sciencedirect.com/science/article/pii/S0042682217301540

6. Wang, L., Yao, W., & Chen, X. (2019). Application of Machine Learning to Identify Influenza Virus Strains. Nature Communications. https://www.nature.com/articles/s41467-019-09560-3

7. Stefanelli, P., Fedele, G., & Ciervo, A. (2020). Identification of Rhinovirus Species Using Random Forest Classifiers. Journal of Clinical Virology. https://www.journalofclinicalvirology.com/article/S1386-6532(20)30241-7/fulltext

8. Kumar, A., Sharma, R., & Gupta, M. (2020). Comparison of Machine Learning Techniques for Identifying Bocavirus in Respiratory Samples. Journal of Medical Virology. https://onlinelibrary.wiley.com/doi/full/10.1002/jmv.25991

9. Zhang, Y., Liu, H., & Xiao, Z. (2019). Ensemble Learning Methods for Detecting Rhinovirus Infections. Bioinformatics. https://academic.oup.com/bioinformatics/article/35/5/841/5100937

10. Xu, Z., Lu, W., & Chen, L. (2021). Deep Learning for Rapid Identification of Respiratory Viruses Including Influenza and Adenovirus. Nature Machine Intelligence. https://www.nature.com/articles/s41598-021-84540-1

11. Mohamed, M., Ali, H., & Zhang, L. (2019). Identification of Human Rhinovirus Genotypes Using Support Vector Machines. Computational Biology and Chemistry. https://www.sciencedirect.com/science/article/abs/pii/S1476927118301891

12. Pereira, H., Santos, F., & Gouveia, R. (2020). A Novel Approach for Classifying Human Bocavirus Genotypes Using Neural Networks. Journal of Clinical Microbiology. https://jcm.asm.org/content/early/2020/07/13/JCM.00932-20

13. Gonzalez-Reiche, A. S., Garcia, J., & Estevez, M. (2019). Predictive Models for Influenza A Subtypes Using Genome-Wide Features and Machine Learning. Journal of General Virology. https://www.microbiologyresearch.org/content/journal/jgv/10.1099/jgv.0.001366

14. Sato, H., Ono, K., & Hashimoto, M. (2020). Utilizing Machine Learning to Identify Adenovirus Serotypes from Genomic Data. Virology Journal. https://virologyj.biomedcentral.com/articles/10.1186/s12985-020-01432-1

15. Lee, C., Park, Y., & Kim, J. (2021). Machine Learning Models for the Differentiation of Respiratory Viruses Including Influenza and Rhinovirus. Microbial Genomics. https://microbegenomics.biomedcentral.com/articles/10.1186/s12864-021-07757-6

16. Park, H., Choi, J., & Kang, H. (2019). Applying Machine Learning to Metagenomic Data for Identifying Bocavirus in Environmental Samples. Applied and Environmental Microbiology. https://aem.asm.org/content/early/2019/10/21/AEM.01989-19

17. Lu, Y., Wei, Z., & Lin, J. (2020). Machine Learning-Based Detection of Rhinovirus in Pediatric Patients. Frontiers in Pediatrics. https://www.frontiersin.org/articles/10.3389/fped.2020.00238/full

18. Jiang, S., Liu, M., & Yang, Q. (2021). Predicting Influenza Virus Evolution with Neural Networks. PLoS Pathogens. https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1008829

19. Palma, D., Martinez, E., & Ferrero, F. (2020). Machine Learning Approaches for Identifying Respiratory Syncytial Virus and Other Respiratory Viruses. Journal of Infectious Diseases. https://academic.oup.com/jid/article/222/4/555/5860133

20. Draper, H., Stewart, R., & Lewis, M. (2019). Using Machine Learning for Virus Typing in Human Adenovirus Infections. Virology Journal. https://virologyj.biomedcentral.com/articles/10.1186/s12985-019-1189-9

21. Altman, D. G., & Bland, J. M. (1995). Statistical tests: The two-sample t-test. BMJ, 310(6983), 170. https://www.bmj.com/content/310/6983/170

22. McHugh, M. L. (2013). The Chi-square test of independence. Biochemia Medica, 23(2), 143-149. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3900058/

23. Hosmer, D. W., Lemeshow, S., & Sturdivant, R. X. (2013). Applied Logistic Regression (3rd ed.). Wiley. https://www.wiley.com/en-us/Applied+Logistic+Regression%2C+3rd+Edition-p-9780470582473

24. Sheskin, D. J. (2020). Handbook of Parametric and Nonparametric Statistical Procedures (5th ed.). CRC Press. https://www.crcpress.com/Handbook-of-Parametric-and-Nonparametric-Statistical-Procedures/Sheskin/p/book/9780367397444

25. Montgomery, D. C., & Runger, G. C. (2010). Applied Statistics and Probability for Engineers (5th ed.). Wiley. https://www.wiley.com/en-us/Applied+Statistics+and+Probability+for+Engineers%2C+5th+Edition-p-9780470053041

26. Wilcoxon, F. (1945). Individual comparisons by ranking methods. Biometrics Bulletin, 1(6), 80-83. https://www.jstor.org/stable/3001968

27. Mood, A. M., Graybill, F. A., & Boes, D. C. (1974). Introduction to the Theory of Statistics (3rd ed.). McGraw-Hill. https://www.mheducation.com/highered/product/introduction-theory-statistics-mood-graybill/

28. Mann, H. B., & Whitney, D. R. (1947). On a test of whether one of two random variables is stochastically larger than the other. The Annals of Mathematical Statistics, 18(1), 50-60. https://projecteuclid.org/euclid.aoms/1177730491

29. Box, G. E. P., & Cox, D. R. (1964). An analysis of transformations. Journal of the Royal Statistical Society: Series B (Methodological), 26(2), 211-243. https://rss.onlinelibrary.wiley.com/doi/10.1111/j.2517-6161.1964.tb00553.x

30. Hollander, M., Wolfe, D. A., & Chicken, E. (2013). Nonparametric Statistical Methods (3rd ed.). Wiley. https://www.wiley.com/en-us/Nonparametric+Statistical+Methods%2C+3rd+Edition-p-9780470387375

Presentations at conferences on the topic of research
================================================
1. 

Software modules developed as part of the study
======================================================
1. A python package *mylib* with all implementation `here .
2. A code with all experiment visualisation `here 
